fix: pubsub configuration (#404)
libp2p/js-libp2p: examples/pubsub/README.md

```
> node 1.js
connected to QmWpvkKm6qHLhoxpWrTswY6UMNWDyn8hN265Qp9ZYvgS82
node2 received: Bird bird bird, bird is the word!
node1 received: Bird bird bird, bird is the word!
node2 received: Bird bird bird, bird is the word!
node1 received: Bird bird bird, bird is the word!
```

> You can change the pubsub `emitSelf` option if you don't want that the publishing node also receives its own messages.

Reviewer: ```suggestion
You can change the pubsub `emitSelf` option if you don't want the publishing node to receive its own messages.
```
